CONTRIBUTING.md

```diff
@@ -110,14 +112,11 @@ PR. That would at least get us started on creating the failing test in the code.
 
 ## Running Tests
 
-There are 3 ways you can run the tests.
-
 * via command-line by running `npm test`
 
-* in the browser by opening `SpecRunner.html`
+* in the browser by opening `test/runner.html`
```

Comment:
It would be sorta slick to add `npm run test-browser` which would run the gulp to build the assets and then afterwards run something like 
```js
const opn = require('opn');

opn('./test/runner.html');
```